Sources/Controllers/2.0/OAOsmEditingViewController.mm

```diff
@@ -318,9 +318,8 @@ + (void) savePoi:(NSString *) comment poiData:(OAEditPOIData *)poiData editingUt
     else
         return;
     [poiData.getTagValues enumerateKeysAndObjectsUsingBlock:^(NSString * _Nonnull key, NSString * _Nonnull value, BOOL * _Nonnull stop) {
-        if (key.length > 0 && value.length > 0 && ![key isEqualToString:POI_TYPE_TAG])
+        if (key.length > 0 && value.length > 0 && ![key isEqualToString:POI_TYPE_TAG] && ![key hasPrefix:REMOVE_TAG_PREFIX])
```

Comment:
This is an incorrect fix. Please sync code with EditPoiDialogFragment.java: 501 instead.

Reply:
Also, the fix to this issue is here: EditPOIMenuBuilder.java: 67